flytekit/core/map_task.py

```diff
@@ -230,6 +230,11 @@ def map_task(task_function: PythonFunctionTask, concurrency: int = None, min_suc
         all inputs are processed.
     :param min_success_ratio: If specified, this determines the minimum fraction of total jobs which can complete
         successfully before terminating this task and marking it successful.
+
+    ``with_overrides`` on a map task can be used to set individual map task resource assigment.
```

Comment:
@katrogan, what's the preferred way of assigning resourced to individual map tasks–`TaskMetadata`/`Resources` or `with_overrides`?

Reply:
I believe it's with `with_overrides` but I haven't worked on flytekit in so long I'm not sure if it's changed! cc @wild-endeavor @eapolinario

Reply:
could you try this out? I believe this should work.   calling `map_task(t1)` produces the map task, and then `()` on that should produce a `Node` which should hit https://github.com/flyteorg/flytekit/blob/master/flytekit/core/node.py#L79

Reply:
but note that this is on the node... not the map task itself.  if you use the map task twice, you have to call with overrides twice.

Reply:
let me know when you want a +1

Reply:
`TaskMetadata` works, whereas `Resources` doesn't. I replaced `Resources` with `with_overrides` in the docstring and test case.